packages/backend/src/migration/1593764167709-alterTables.ts

```diff
@@ -8,14 +8,18 @@ export class alterTables1593764167709 implements MigrationInterface {
         await queryRunner.query(`DROP SEQUENCE "modos"."observation_id_seq";`, undefined);
         await queryRunner.query(`ALTER TABLE "modos"."observation" ALTER COLUMN "id" ADD GENERATED ALWAYS AS IDENTITY;`, undefined);
 
-        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "position" geometry(Point, 4326) NOT NULL;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "position" geometry(Point, 4326);`, undefined);
         await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "in_range" BOOLEAN NOT NULL DEFAULT FALSE;`, undefined);
-        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "edist" REAL NOT NULL;`, undefined);
-        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "snap_geom" geometry(Point, 4326) NOT NULL`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "edist" REAL;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "created_at" TIMESTAMP WITH TIME ZONE DEFAULT NOW();`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "updated_at" TIMESTAMP WITH TIME ZONE DEFAULT NOW();`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "snap_geom" geometry(Point, 4326);`, undefined);
 
         await queryRunner.query(`ALTER TABLE "modos"."event" ALTER COLUMN "id" DROP DEFAULT;`, undefined);
         await queryRunner.query(`DROP SEQUENCE "modos"."event_id_seq";`, undefined);
         await queryRunner.query(`ALTER TABLE "modos"."event" ALTER COLUMN "id" ADD GENERATED ALWAYS AS IDENTITY;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."event" ADD COLUMN "created_at" TIMESTAMP WITH TIME ZONE DEFAULT NOW();`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."event" ADD COLUMN "updated_at" TIMESTAMP WITH TIME ZONE DEFAULT NOW();`, undefined);
```

Comment:
If we got a mix of camel and snake case, it the begining of problem. Stick with CamelCase.

Reply:
Fixed in https://github.com/MediaComem/modos/pull/59/commits/f486309c676e9b5b955fd870acae9251c6080455